docs/UserGuide.md

```diff
@@ -168,60 +267,262 @@ Examples:
+
+Format: `deleteproject INDEX`
+
+- Deletes the project at the specified `INDEX`
+- The index refers to the **index number shown in the displayed project list.**
+- The index must be a positive integer 1, 2, 3, …​
+
+Examples:
+
+- `listproject` followed by `deleteproject 2` deletes the **2nd project shown**.
+- `findproject Alpha` followed by `deleteproject 1` deletes the **1st project in the results** of the `findproject` command.
+
+Expected output:
+
+- System message noting success
+- Deleted project vanishes from displayed project list
+
+[Return to Top](#table-of-contents)
+
+### Clear all project entries : `clearproject`
+
+**Clears all project entries** from HRConnect.<br>
+<span style="color:red">This will delete all project records AND their assignments\!</span>
+
+Format: `clearproject`
+
+Expected output:
+- System message noting success
+- Displayed project list becomes empty
+
+[Return to Top](#table-of-contents)
+
+### Create Assignment : `assign`
+
+**Creates an assignment** between a Person and a Project.
+
+Format: `assign aid/ASSIGNMENT_ID pid/PROJECT_ID id/EMPLOYEE_ID`
+
+- The `PROJECT_ID` must belong to an existing project.
+- The `EMPLOYEE_ID` must belong to an existing person.
+
+Examples:
+
+- `assign aid/1 pid/A02 id/1`
+
+Expected output:
+
+- System message noting success and information about assignment added
+
+[Return to Top](#table-of-contents)
+
+### List Assignments : `listassignments`
+
+Shows a list of **all stored assignments**.
+
+Format: `listassignments`
+
+Expected output:
+
+- System message noting success
+- All assignments shown in the top display list
+
+[Return to Top](#table-of-contents)
+
+### Delete Assignment: `unassign`
+
+**Deletes an assignment** between a Person and a Project.<br>
+This does not delete the Person nor the Project.
+
+Format: `unassign aid/ASSIGNMENT_ID`
+
+- The `ASSIGNMENT_ID` must belong to an existing assignment.
+
+Examples:
+
+- `unassign aid/1`
+
+Expected output:
+
+- System message noting the `ASSIGNMENT_ID` of the assignment removed.
+
+[Return to Top](#table-of-contents)
+
 ### Exiting the program : `exit`
 
 Exits the program.
 
 Format: `exit`
 
+[Return to Top](#table-of-contents)
+
 ### Saving the data
 
-AddressBook data are saved in the hard disk automatically after any command that changes the data. There is no need to save manually.
+HRConnect data are saved in the hard disk automatically after any command that changes the data. There is no need to save manually.
+
+[Return to Top](#table-of-contents)
 
 ### Editing the data file
 
-AddressBook data are saved automatically as a JSON file `[JAR file location]/data/addressbook.json`. Advanced users are welcome to update data directly by editing that data file.
+HRConnect data are saved automatically as a JSON file `[JAR file location]/data/addressbook.json`. Advanced users are welcome to update data directly by editing that data file.
```

Comment:
Changed "AddressBook" to "HRConnect". Will create an issue to update the name of the JSON file to `hrconnect.json`.